<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySerialPort</class>
 <widget class="QWidget" name="MySerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(151, 156, 255);
color: rgb(0, 170, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>228</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>编队设备设置</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>32</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DEVICE1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>63</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>DEVICE2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>94</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>DEVICE3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="device1_LineEdit">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="device2_Lineedit">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="device3_LineEdit">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deviceSet_Btn">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>60</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>21</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>发送串口</string>
   </property>
   <widget class="QComboBox" name="SerialPort_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="baud_SerialPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="flushSerial_Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openSerialPort_Btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>70</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>接收串口</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>串口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="receiveSerialPort_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>27</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="baudReceiveSeriaolport_ComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>67</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="flushReceiveSerialPort_Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openReceiveSerialPort_Btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
